src/main/java/de/focusshift/zeiterfassung/integration/urlaubsverwaltung/overtime/OvertimeRabbitmqConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package de.focusshift.zeiterfassung.integration.urlaubsverwaltung.overtime;
+
+
+import de.focusshift.zeiterfassung.tenancy.tenant.TenantContextHolder;
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(value = "zeiterfassung.integration.urlaubsverwaltung.overtime.enabled", havingValue = "true")
+@EnableConfigurationProperties(OvertimeRabbitmqConfigurationProperties.class)
+class OvertimeRabbitmqConfiguration {
+
+    static final String ZEITERFASSUNG_URLAUBSVERWALTUNG_OVERTIME_ENTERED_QUEUE = "zeiterfassung.queue.urlaubsverwaltung.overtime.entered";
```

Comment:
Whats the purpose of this queue? Is it for consumption of applications for overtime reduction of uv? Are they consumed by [`ZEITERFASSUNG_URLAUBSVERWALTUNG_APPLICATION_ALLOWED_QUEUE`](https://github.com/urlaubsverwaltung/zeiterfassung/blob/2e578bfea3e491fa004e82f6057dc372ee1271ce/src/main/java/de/focusshift/zeiterfassung/integration/urlaubsverwaltung/application/ApplicationRabbitmqConfiguration.java#L46-L46)?